File: Practica-POO/Semana01/Ejercicio07/Employ.cpp
#include <iostream>
#include <string>
using namespace std;

class Employ{
    public:
    string name;
    double salary;

    Employ(string n , double s){
        name = n;
        salary = s;
    }

    double Discount(){
        return salary*0.1;
    }

    double NetSalary(){
        return salary-Discount();
    }
};

int main(){
    Employ employ("Jeremy", 12500);

    double discount1 = employ.Discount();
    double netsalary1 = employ.NetSalary();
    cout << "El descuento es de: " << discount1 << endl;
    cout << "El salario neto es de: " << netsalary1 << endl;
    
    return 0;
}

File: Practica-POO/Semana01/Ejercicio07/Employ.py
class Employ:
    def __init__(self,name,salary):
        self.name = name 
        self.salary = salary

    def Discount(self):
        return self.salary*0.1

    def NetSalary(self):
        return self.salary - self.Discount()


employ1 = Employ("Jeremy",7500)

discount1 = employ1.Discount()
netsalary1 = employ1.NetSalary()

print(f"El descuento es de: {discount1}")
print(f"El salario neto es: {netsalary1}")